test/library/packages/DistributedBag/nqueens.compopts

```diff
@@ -0,0 +1 @@
+--fast
```

Comment:
Why is `--fast` added here? In our nightly testing, we have different configurations that test with and without `--fast`. Is this test too slow to be practical if `--fast` is not given?

Reply:
I think that 28s is acceptable & we can add intermediate flags like `-O` later if it becomes too slow. I think it's important that it run with bounds checking on in some configurations, and `--fast` disables bounds checking.